incubator/busbar/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+name: busbar
+version: 0.0.1
+appVersion: 1.5.0
+description: A tool to build, deploy and control applications running on a Kubernetes cluster.
+keywords:
+- application hosting
+home: https://busbar.io
+sources:
+- https://github.com/busbar-io/busbar-server
+maintainers:
+- name: busbar.io
```

Comment:
For a maintainers name we're looking for a GitHub ID. This way we have someone to notify when a PR or change comes it.